packages/core-flows/src/handlers/product/upsert-variant-prices.ts

```diff
@@ -95,7 +95,9 @@ export async function upsertVariantPrices({
           min_quantity: price.min_quantity,
           max_quantity: price.max_quantity,
           amount: price.amount,
-          currency_code: region_currency_code ?? price.currency_code,
+          currency_code: (
+            region_currency_code ?? price.currency_code
+          ).toLowerCase(),
```

Comment:
question: is this required or is it because of the export data? 

we didn't have this issue previously